### PR TITLE
kubevirt: Add VM Dashboard Inventory Card

### PR DESCRIPTION
**Pull request by mareklibra**
On the VM Details Overview tab.

Depends on:
- [x] #2600 
- [x] #2601

Just the last commit is relevant.

### CODE REVIEW

**Comment by mareklibra** [comment]
vmInventory

**Comment by mareklibra** [comment]
cc @andybraren

**Comment by mareklibra** [comment]
Rebased to `master-4.3`

@rawagner, can you please have a look?

**Comment by mareklibra** [comment]
Rebased as #2600 is merged.

File: frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts [hunk 1]
```diff
@@ -10,3 +11,6 @@ export const getVMIApiPath = (vmi: VMIKind) =>
   `${VirtualMachineInstanceModel.apiVersion}/namespaces/${getNamespace(vmi)}/${
     VirtualMachineInstanceModel.plural
   }/${getName(vmi)}`;
+
+export const getVMINetworks = (vmi: VMIKind) => _.get(vmi, 'spec.networks', []);
```
Comment (rawagner) [inline frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts]: please add return type ie `VMIKind['spec]['networks']`, same for `getVMIDisks`

Comment (mareklibra) [reply frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts]: Per discussion in other PR, I have replaced the `_.get()` by direct access.

Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts]: nice

**Comment by mareklibra** [comment]
Rebased, #2601 is merged now

**Comment by rawagner** [comment]
tests are failing

/hold

**Pull request merged**